pkg/api/transform_network.go

```diff
@@ -24,6 +24,18 @@ type TransformNetwork struct {
 	ProtocolsFile  string                `yaml:"protocolsFile,omitempty" json:"protocolsFile,omitempty" doc:"path to protocols file (optional, default: /etc/protocols)"`
 }
 
+func (tn *TransformNetwork) GetServiceFiles() (string, string) {
+	p := tn.ProtocolsFile
+	if p == "" {
+		p = "/etc/protocols"
```

Comment:
by the way, I used the previous default, as you can see here: https://github.com/netobserv/flowlogs-pipeline/pull/304/files#diff-803cd3d555b315c0ffdf0ac4905fd947085c6edf178137e7734e7af8a1218214L55

But maybe this isn't super safe to write in /etc/protocols? maybe rather use /tmp/flp-protocols and /tmp/flp-services instead? @KalmanMeth @ronensc @mariomac  ?

Reply:
never mind, I thought it was a file that we would write into, I didn't know it was a standardized thingy